src/utils/Firebase.tsx

```diff
@@ -42,10 +42,34 @@ export async function getUserDataForSpecificTask(jobID: string) {
     console.log(userDoc.data());
 }
 
+export async function createNewJob(userID: string, title: string, description: string, jobSkills: string[]) {
+    try {
+        const newJob = {
+            userID: userID,
+            title: title,
+            description: description,
+            jobSkills: jobSkills,
+            createdAt: new Date()
+        };
+        
+        const docRef = await addDoc(collection(db, "Jobs"), newJob);
+        const returnedJob = await getDoc(docRef);
+
+        if (returnedJob.exists()) {
+            console.log("Job created with ID: ", docRef.id);
+            return { id: docRef.id, ...returnedJob.data() };
+        } else {
```

Comment:
if the docRef fails it should error out of the try block anyways so you would probably never get this this else code.